web/src/elements/forms/SearchSelect/ak-search-select-view.ts

```diff
@@ -264,21 +265,6 @@ export class SearchSelectView extends AKElement {
     }
 }
 
-type Pair = [string, string];
```

Comment:
eslint/no-use-before-define

Moving this into a `./utils.ts` folder helped, and helped keep this cleaner.